### PR TITLE
Updated about Cartographer in report

### PR DESCRIPTION
**Pull request by linwil-5**

### CODE REVIEW

**Review by anti002 — CHANGES_REQUESTED** [review]
Some of it should go in discussion instead, ex. last part of first paragraph

**Review by anti002 — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -84,9 +84,14 @@ The LIDAR was used to gather all the data used to map the environment. The raw d
 
 ### Cartographer
 <p>
-Cartographer is an algorithm/system used to get a representation of the local environment given useful sensor data. For localization and mapping of the local environment, SLAM is the main part. 
+The Cartographer system is derived from Google’s repository in GitHub where the system is an open source project, letting us to use it. This system’s main focus is to be a solution to the simultaneous localization and mapping (SLAM) problem that is about how one can do both localization and mapping at the same time.
```
Comment (zachtyll) [inline README.md]: This part needs rephrasing. "Letting us to use it." is not proper grammar.

File: README.md [hunk 2]
```diff
@@ -84,9 +84,14 @@ The LIDAR was used to gather all the data used to map the environment. The raw d
...
+
+The usefulness of this system is in the quality of representing the environment, high resolution maps can be easily created with this system. Cartographer itself creates a compressed protobuf file (pbstream) as an output which only contains what Cartographer claims to be useful sensor data and then when recombined with full sensor data file (bag) creating a high resolution map. Although it's in our interest to combine multiple maps from multiple robots into one map, there was no clear way to do that with the system. Instead the output file in interest became a portable grey map (PGM) file. For Cartographer to work with the rest of the parts, necessary tuning on the system’s parameters had to be done to work with the turtlebot3.
```
Comment (zachtyll) [inline README.md]: Change "high resolution" to "high-resolution".

Add "," after "Instead the output file..."

File: README.md [hunk 3]
```diff
@@ -113,6 +118,7 @@ In this case, Central station is the ROS master. Its role is to look over the ot
 
 ## Discussion
 <p>
+As Cartographer is an approach to this dilemma, different approaches such as the Hector SLAM approach and Gmapping SLAM could have been tested. Due to the amount of time consumed to get to understand, implement and tune Cartographer, there would be insufficient time to do the rest.
```
Comment (zachtyll) [inline README.md:121]: Add references to other approaches if we do not describe them in text.

**Review by zachtyll — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (linwil-5) [reply README.md]: Ok

**Review by zachtyll — APPROVED** [review]

**Review by anti002 — APPROVED** [review]

**Pull request merged**